docker/server/.env.example

```
@@ -25,3 +25,6 @@ HEXCLAVE_FREESTYLE_API_KEY=
 HEXCLAVE_CLICKHOUSE_URL=http://host.docker.internal:8133
 HEXCLAVE_CLICKHOUSE_ADMIN_PASSWORD=password
 HEXCLAVE_CLICKHOUSE_EXTERNAL_PASSWORD=password
+HEXCLAVE_CLICKHOUSE_PUBLIC_HOST=localhost
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HEXCLAVE_CLICKHOUSE_PUBLIC_HOST key should go before the HEXCLAVE_CLICKHOUSE_URL key

(UnorderedKey)